fdbcli/TenantCommands.actor.cpp

```diff
@@ -273,6 +273,43 @@ ACTOR Future<bool> tenantDeleteCommand(Reference<IDatabase> db, std::vector<Stri
 	return true;
 }
 
+// tenant deleteID command
+ACTOR Future<bool> tenantDeleteIdCommand(Reference<IDatabase> db, std::vector<StringRef> tokens) {
+	if (tokens.size() != 3) {
+		fmt::print("Usage: tenant deleteId <ID>\n\n");
+		fmt::print("Deletes a tenant from the cluster by ID.\n");
+		fmt::print("Deletion will be allowed only if the specified tenant contains no data.\n");
+		return false;
+	}
+	state Reference<ITransaction> tr = db->createTransaction();
+	loop {
+		try {
+			tr->setOption(FDBTransactionOptions::SPECIAL_KEY_SPACE_ENABLE_WRITES);
+			tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+			state ClusterType clusterType = wait(TenantAPI::getClusterType(tr));
+			state int64_t tenantId = std::stoi(tokens[2].toString());
```

Comment:
This needs to some more input validation and handle errors. You could switch to using `sscanf` in the way that other fdbcli commands do, or you could also make it work with `stoi`.